### ISSUE TITLE
when i trained unet model,the accuracy is 1 and the loss is very low.

### ISSUE THREAD

**Issue by AtomLebron** [divamgupta/image-segmentation-keras#72]
the last,my predicted images  are  all bule,can u tell me what should i do?thanks a lot!By the way,i used my own data.My data use the labelme to annotate.

**Comment by AtomLebron** [comment]
when i try to use vgg_unet,the accuracy still is 1.

**Comment by AtomLebron** [comment]
Then i try to use author's dataset1 to train vgg_unet,and the output predicted image is ok,and the accuracy is ok,so i think my own data is error.
But i don't know how to do that For each pixel in the RGB image, the class label of that pixel in the annotation image would be the value of the blue pixel.what's the code,who can tell me that how did you prepare your data,thanks a lot!

**Comment by AtomLebron** [comment]
i have sloved the problem about accuracy,because the labeled image are gray,but my labeled images are RGB,but the output images only have two kind of colours.Who can tell me why?Thanks  a lot.And when i try to set n_classes to 2,it will raise error,but my data only have background and one target.

**Comment by divamgupta** [comment]
In your case , you should set the pixel of the background as (0,0,0) and target as (1,1,1)

**Comment by AtomLebron** [comment]
@divamgupta thank u a lot.can this project output image like maskrcnn'output？

**Comment by divamgupta** [comment]
Mask-rcnn does detection along with segmentation. On the other hand, we only do semantic segmentation. Hence this project cannot produce bounding boxes.
